witness/src/Dscp/Snowdrop/PublicationValidation.hs

```diff
@@ -0,0 +1,106 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Dscp.Snowdrop.PublicationValidation
+       ( validatePublication
+       , Publication (..)
```

Comment:
Wait what? Reexports again? Please stop doing that.

Reply:
Will remove. Forgot, when I moved it to `Dscp.Core`